.github/workflows/build-interop-docker.yml

```
@@ -19,9 +25,14 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: set tag name
+        if: github.event_name == 'tag'
```

The event_name would be push for either tag or branch push.

We could replace the check with `startsWith(github.ref, 'refs/tags/')` but I suggest removing it altogether and deciding on the tag using the step logic.

```suggestion
        id: tag
```